src/olympia/addons/tasks.py

```diff
@@ -562,6 +565,21 @@ def add_static_theme_from_lwt(lwt):
     for addon_tag in AddonTag.objects.filter(addon=lwt):
         AddonTag.objects.create(addon=addon, tag=addon_tag.tag)
 
+    # Steal the ratings (even with soft delete they'll be deleted anyway)
+    addon_updates.update(
+        average_rating=lwt.average_rating,
+        bayesian_rating=lwt.bayesian_rating,
+        total_ratings=lwt.total_ratings,
+        text_ratings_count=lwt.text_ratings_count)
+    rating_activity_log_ids = [
+        l.id for l in amo.LOG if getattr(l, 'action_class', '') == 'review']
+    for rating in Rating.unfiltered.filter(addon=lwt):
+        rating.update(addon=addon, version=version)
+    # Modify the activity log entry too.
+    for alog in AddonLog.objects.filter(
+            addon=lwt, activity_log__action__in=rating_activity_log_ids):
```

Comment:
Maybe use `.iter()` on that queryset to avoid loading a bunch of `AddonLog` instances in memory at once. There could be a lot of them in some cases...

Reply:
https://github.com/mozilla/addons-server/pull/8403/commits/4996e950d935899a318c14c5dc3ce5bed43a37f1